Karma/Karma.csproj

```diff
@@ -35,7 +35,7 @@
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
       <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="5.0.11" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.11" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.12" />
```

Comment:
You didn't mention anything about updating database version in PR summary. Is it supposed to be updated?

Reply:
Yeah, propbaby happened when I ran `update-database`. Although, I don't know if it makes any difference, does it?